### PR TITLE
[TA] test live tests in private endpoint

### PR DESCRIPTION
**Pull request by mssfang**
Modified current existing testResources.json and tests.yml files to enable the live test in private endpoint.

### CODE REVIEW

File: sdk/textanalytics/tests.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 trigger: none
 
+resources:
```
Comment (maririos) [inline sdk/textanalytics/tests.yml:3]: why is this needed?

**Review by samvaity — APPROVED** [review]
As far as we roll this back when service endpoint is public, lgtm

File: sdk/textanalytics/tests.yml (hunk 1 above)
Comment (mssfang) [reply sdk/textanalytics/tests.yml:3]: It should not added. I was looking at the .NET's older commint (through the Blame https://github.com/Azure/azure-sdk-for-net/blame/3c6a978b247cce2f071dfa57db2288fb0882e8ca/sdk/textanalytics/tests.yml),

**Review by danieljurek — COMMENTED** [review]
Rather than edit the test-resources.json which might require reverting later, set `DisableAzureResourceCreation: true` in the template parameters. That will disable resource deployment for the job, save agent time, and save deploying an unused resource. 

When the service GA's appropriately you can revert the changes to tests.yml and you'll have the old way of doing things back. 


```yaml
trigger: none

jobs:
  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
    parameters:
      ServiceDirectory: textanalytics
      Artifacts:
        - name: azure-ai-textanalytics
          groupId: com.azure
          safeName: azureaitextanalytics

      # Add the below line and some note describing why resource creation is disabled
      DisableAzureResourceCreation: true
      EnvVars:
          AZURE_TEXT_ANALYTICS_API_KEY: $(java-textanalytics-test-api-key)
          AZURE_TEXT_ANALYTICS_ENDPOINT: $(java-textanalytics-test-ppe-endpoint-string)
```
